win32ss/user/ntuser/object.c

```diff
@@ -871,3 +871,22 @@ PVOID FASTCALL UserAssignmentUnlock(PVOID *ppvObj)
 
     return pvOld;
 }
+
+VOID FASTCALL UserThreadLock(PTL pTL, PVOID pobj)
```

Comment:
It would be of interest to compare those two new functions with those ReactOS-specific User(De)refObjectCo ones:
https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/ntuser/object.h;hb=6d2012ac3b4dddb0f19fff79ccc9738709979715#l26